test/extended/util/nfs.go

```diff
@@ -0,0 +1,184 @@
+package util
+
+import (
+	"fmt"
+	"time"
+
+	kapiv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kutilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+// CreateNFSServerReplicationController creates an nfs server replication controller
+func CreateNFSServerReplicationController(name, capacity string) *kapiv1.ReplicationController {
+	replicas := int32(1)
+	privileged := true
+
+	return &kapiv1.ReplicationController{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ReplicationController",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   name,
+			Labels: map[string]string{"name": name},
+		},
+		Spec: kapiv1.ReplicationControllerSpec{
+			Replicas: &replicas,
+			Selector: map[string]string{
+				"role": "nfs-server",
+			},
+			Template: &kapiv1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"role": "nfs-server"},
+				},
+				Spec: kapiv1.PodSpec{
+					Containers: []kapiv1.Container{
+						{
+							Name: "nfs-server",
+							// The code for this image is located at https://github.com/coreydaley/nfs-server
+							// The image exports 10 mounts, /exports/data-0 through /exports/data-9
+							Image: "docker.io/coreydaley/nfs-server",
```

Comment:
Is this the long term destination for this image, or do we have plans to duplicate this image, but have it under somewhere besides `coreydaley` ?

Pretty sure in the past an effort was made to move test artifacts out of individual developer's repos

Reply:
I think that the ultimate goal will be to have a Dockerfile in the origin repo that will build this image and run it.

Reply:
yeah that is the ultimate goal.